lib/chutney/TorNet.py

```diff
@@ -1323,11 +1358,12 @@ def getNodeDirInfoStatusPattern(self, dir_format):
         elif desc:
             return r'^router ' + nickname + " "
         elif md:
```

Comment:
I think we are only getting None for ed25519_key here. Because we are loading ed25519_id during configure(). But this code is called during wait_for_bootstrap().

Can you help me check what is happening?

If md is True, print the values of nickname and ed25519_key. Then we can check the logs.